packages/ocean-lending/integration-tests/oceanGovernor.ts

```diff
@@ -125,18 +126,19 @@ describe('OceanGovernor', () => {
     // Get proposal detail
     const proposalDetail = await admin.getProposalDetail(hash);
 
-    const expectPool = JSON.stringify(pool, (_, v) => typeof v === 'bigint' ? v.toString() : v);
-    const actualPool = JSON.stringify(proposalDetail.pool, (_, v) => typeof v === 'bigint' ? v.toString() : v)
+    const expectPool = pool;
+    const actualPool = proposalDetail.pool;
+    const expectPoolStr = JSON.stringify(expectPool, (_, v) => typeof v === 'bigint' ? v.toString() : v);
+    const actualPoolStr = JSON.stringify(actualPool, (_, v) => typeof v === 'bigint' ? v.toString() : v)
 
-    // assert expected are equal
+    // Assert expected and actual proposal are equal
     ensure(
       proposalDetail.leasePeriod.toString() == leasePerod &&
-      expectPool == actualPool &&
+      _.isEqual(actualPool, expectPool) &&
```

Comment:
this is not the right way to do it. Should import `isEqual` directly instead of as _. Refer to https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark